Documentation/building/linux-instructions.md

```diff
@@ -71,7 +109,7 @@ If you are using Fedora, then you will need to install the following packages:
 Git Setup
 ---------
 
-This guide assumes that you've cloned the corefx and coreclr repositories into `~/git/corefx` and `~/git/coreclr` on your Linux machine. If your setup is different, you'll need to pay careful attention to the commands you run. In this guide, I'll always show what directory I'm in.
+This guide assumes that you've cloned the coreclr repositories and are at the root of the coreclr folder.
```

Comment:
It's only one repository -- namely _coreclr_ -- so `cloned the coreclr repository`.